### ISSUE TITLE
Errno 1

### ISSUE THREAD

**Issue by SAMURAIFISSHU** [JunkFood02/Seal#1671]
### Checklist

- [ ] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

Seal doesn't let me download any audio or video 

### Error reports

App version: 1.13.0 (11302)
Device information: Android 12 (API 31)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2024.09.02.232855

URL: https://youtu.be/T_gChlxSdSk?si=I9SwDySwiRJVQyEb
ERROR: [Errno 1] Operation not permitted: '/storage/emulated/0/Download/Seal/tmp/＂Your Power＂ (cover) by Billie Eilish [T_gChlxSdSk].webp'

**Comment by AnonymousDab84** [comment]
Yes since the latest update I get the same error on any video or audio.

**Comment by SAMURAIFISSHU** [comment]
> Yes since the latest update I get the same error on any video or audio.  

You found a solution?
